src/polytopes/chain.jl

```diff
@@ -15,6 +15,9 @@ end
 Chain(vertices::CircularVector) =
   Chain([collect(vertices); vertices[begin]])
 
+Chain(vertices::AbstractVector) = Chain(collect(vertices))
```

Comment:
I wasn't sure whether to juxtapose it to the previous definition, but I think it's more readable this way.

Reply:
I wonder if we need this one explicitly given that `Polytope` already defines a constructor for AbstractVector. Is it am ambiguity issue?

Reply:
`Polytope` defines this constructor for vectors:
```julia
(::Type{PL})(vertices::AbstractVector{TP}) where {PL<:Polytope,TP<:Tuple} = PL(Point.(vertices))
```
But it won't work for `AbstractVector{<:Point}`, it works only for tuples. Would it be better maybe to extend to above `Chain` constructor for all polytopes?

Reply:
The problem is that certain polytopes may have a `vertices` field that is a `Vector` (like `Chain`s), or `AbstractVector` which may be `SVector`s or not. Unless we change `Chain` to parametrize its vector type, we will have to redefine a special constructor for `Chain`.

Reply:
Yes, I think I tried adding this constructor to all polytopes at some point but it caused ambiguities down the line. Can you give it a try? I think that was the reason why I didn't add these constructors to Chain so far. Ideally we would be able to have them in all Polytope.

Reply:
We could either
- assume that all polytopes will prefer `SVector`s
- have polytopes that don't want a `SVector` redefine their own constructor
or
- assume that all polytopes will prefer `Vector`
- have polytopes that don't want a `Vector` redefine their own constructor
or
- make no assumptions on the preferred vector type
- have all polytopes rely on their internal constructor
- have all polytopes redefine their own version of the definition below (which would be removed)

```julia
(::Type{PL})(vertices::Vararg{P}) where {PL<:Polytope,P<:Point} = PL(SVector(vertices))
```

Reply:
Taking the time to process this important design decision. I understand from your bullet points that we have 3 main alternative designs, correct? Could you please group them differently so that we can reason more clearly about it. :)

Reply:
Oh I see the "or" you typed there now. Nevermind.

Reply:
It's actually a bit more nasty. I'll make a few experiments and report the possibilities. Feel free to do the same if you are at ease with constructor shenanigans.

Reply:
Thanks for experimenting with these ideas. I am not very good at constructor shenanigans so I am glad you can help on this front :)